opendatahub-io/odh-manifests
# WIP: Update Spark Operator to support ocp 4.9

Replaces #506 and adds changes to resolve issues with new OCP 4.9 requirements

Instructions to test:
* Deploy ODH with JupterHub and Spark operator enabled
* Confirm JupyterHub and Spark operator pods are running successfully
* Login to JupyterHub and Spawn a spark notebook.
* Confirm that the notebook spawns successfully and a user spark cluster pods are deployed
